### PR TITLE
New format and uniform softmax

### PR DESCRIPTION
**Pull request by mirkobronzi**

### CODE REVIEW

File: bert_reranker/main.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 import numpy as np
 import pytorch_lightning as pl
 import torch
+import torch.multiprocessing
```
Comment (jerpint) [inline bert_reranker/main.py:12]: do we still need this?

File: bert_reranker/main.py [hunk 2]
```diff
@@ -82,7 +83,7 @@ def main():
         action="store_true",
     )
     parser.add_argument(
-        "--num_workers", help="number of workers - default 2", type=int, default=2
+        "--num-workers", help="number of workers - default 2", type=int, default=2
```
Comment (jerpint) [inline bert_reranker/main.py:86]: should default be set to 0 now?

File: bert_reranker/main.py [hunk 3]
```diff
@@ -102,6 +103,9 @@ def main():
 
     logging.basicConfig(level=logging.INFO)
 
+    # Fix for num_workers > 0
+    # torch.multiprocessing.set_sharing_strategy('file_system')
+
```
Comment (jerpint) [inline bert_reranker/main.py:108]: these can be removed?
